### PR TITLE
antialias=True 

### PR DESCRIPTION
**Pull request by jere357**
addressing  https://github.com/sayakpaul/cmmd-pytorch/issues/4
the torch.nn.functional.interpolate **defaults antialias to False**, while the jax version defaults it to True,

the align_corners in torch also defaults to False.

[torch.nn.functional.interpolate docs](https://docs.pytorch.org/docs/stable/generated/torch.nn.functional.interpolate.html)
[jax.image.resize docs](https://docs.jax.dev/en/latest/_autosummary/jax.image.resize.html)

### CODE REVIEW

File: embedding.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
 
 def _resize_bicubic(images, size):
     images = torch.from_numpy(images.transpose(0, 3, 1, 2))
-    images = torch.nn.functional.interpolate(images, size=(size, size), mode="bicubic")
+    images = torch.nn.functional.interpolate(images, size=(size, size), mode="bicubic", align_corners=False, antialias=True)
```
Comment (sayakpaul) [inline embedding.py:28]: Does `align_corner` also need to be specified?

**Review by sayakpaul — COMMENTED** [review]
Thanks for your PR. Can we also compute the metric to see if it deviates from the original?
